apps/studio/components/interfaces/TableGridEditor/EmptyState.tsx

```diff
@@ -29,6 +30,7 @@ const EmptyState = ({}: EmptyStateProps) => {
     connectionString: project?.connectionString,
     schemas: [selectedSchema],
     sort,
+    filterTypes: Object.values(ENTITY_TYPE),
```

Comment:
maybe this can be a default value on useEntityTypesQuery - i realised we sort of do this here
but filterTypes should have been defaulted in the prop declaration instead
![image](https://github.com/user-attachments/assets/1b6bd086-7a70-4e11-9d19-46add3a6a7b8)